src/PageCache.php

```diff
@@ -11,15 +11,15 @@
 
 namespace PageCache;
 
-use PageCache\Storage\FileSystem\FileSystemCacheAdapter;
-use PageCache\Strategy\DefaultStrategy;
+use DateTime;
 use PageCache\Storage\CacheItem;
 use PageCache\Storage\CacheItemInterface;
 use PageCache\Storage\CacheItemStorage;
+use PageCache\Storage\FileSystem\FileSystemCacheAdapter;
```

Comment:
if we are changing order here, please let's keep it in alphabetical order.

Reply:
Sorry, they are (PhpStorm sort them automatically).